server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/ThreadGetContract.scala

```diff
@@ -165,6 +165,7 @@ trait ThreadGetContract {
         MessageManager.AppendCommand.from(Message.Builder.of.setSubject("Test")
         .setMessageId("Message-ID-1")
           .setBody("testmail", StandardCharsets.UTF_8)))
+    Thread.sleep(1000)
```

Comment:
I prefer to not use the .sleep and wrap the assertion in an await.

`Await.untilAsserted(() -> ...)`